app/Jobs/ImportFacebookSharePosts.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace Chompy\Jobs;
+
+use League\Csv\Reader;
+use Chompy\Services\Rogue;
+use Illuminate\Bus\Queueable;
+use Chompy\Events\LogProgress;
+use Chompy\Traits\ImportToRogue;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Support\Facades\Storage;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+
+class ImportFacebookSharePosts implements ShouldQueue
+{
+    use Dispatchable, InteractsWithQueue, Queueable, SerializesModels, ImportToRogue;
+
+    /**
+     * The path to the stored csv.
+     *
+     * @var string
+     */
+    protected $filepath;
+
+    /**
+     * Stat counter.
+     */
+    protected $stats;
+
+    /**
+     * Total record counter.
+     */
+    protected $totalRecords;
+
+    /**
+     * Create a new job instance.
+     *
+     * @return void
+     */
+    public function __construct($filepath)
+    {
+        $this->filepath = $filepath;
+
+        $this->stats = $this->statsInit();
```

Comment:
@chloealee Hmmm now I'm wondering if this shouldn't be part of a trait and should be per import, because 2 of the 5 stats do not apply to this import, which doesn't hurt anything but they'll just sit at 0. What do you think?

Reply:
@katiecrane that is a good call! I don't have strong opinions but I don't think it hurts for them to sit at 0, mostly because that is still an accurate count (0 records needed to be scrubbed, 0 accounts needed to be made)?

Reply:
@chloealee perfect! that's super reasonable to me!